config.yaml

```diff
@@ -49,3 +49,19 @@ options:
     default: ""
     description: |
       A YAML list which will be injected to define additional prospectors/inputs.
+  ignore_older:
+    type: string
+    default: "48h"
+    description: |
+      If this option is enabled, Filebeat ignores any files that were modified before the specified timespan. 
+      Configuring ignore_older can be especially useful if you keep log files for a long time. Allowed values
+      are valid integers (no leading zeroes) followed by either 'h' (hours) or 'm' (minutes), e.g., "5m", "24h":
+      https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-log.html#filebeat-input-log-ignore-older"
+  clean_inactive:
+    type: string
+    default: "72h"
+    description: |
+      When this option is enabled, Filebeat removes the state of a file after the specified period of inactivity has elapsed:
+      Allowed values are valid integers (no leading zeroes) followed by either 'h' (hours) or 'm' (minutes), e.g., "5m", 
```

Comment:
advice: you could also make your life a hell of a lot easier by only accepting int with the meaning of minutes. All of your validation functions would almost disappear

Reply:
correcting myself, I fully agree accepting only integers (in seconds for better granularity) simplifies the code quite a lot. Refactoring and testing